README.md

```diff
@@ -145,6 +145,16 @@ This image is configured with volumes at `/data` and `/datalog` to hold the Zook
 
 > Be careful where you put the transaction log. A dedicated transaction log device is key to consistent good performance. Putting the log on a busy device will adversely affect performance.
 
+## How to configure logging
+
+By default, ZooKeeper redirects stdout/stderr outputs to the console. You can redirect to a file located in /logs by passing environment variable ZOO_LOG4J_PROP as follows:
+
+    $ docker run --name some-zookeeper --restart always -d -v -e ZOO_LOG4J_PROP="INFO,ROLLINGFILE" 31z4/zookeeper
```

Comment:
This command is incorrect. It gives the following error

    docker: invalid reference format: repository name must be lowercase.

because `-v` requires an argument. Also `-d` is not requred here. So let's just go with

    $ docker run --name some-zookeeper --restart always -e ZOO_LOG4J_PROP="INFO,ROLLINGFILE" 31z4/zookeeper